dialect/sql/schema/sqlite.go

```diff
@@ -6,6 +6,7 @@ package schema
 
 import (
 	"context"
```

Comment:
Let’s stay consistent with the current convention: https://github.com/ent/ent/blob/2c63d1d70e6c9ed7648abd1fbfdbfc5a84ff5362/entc/integration/integration_test.go#L9